### PR TITLE
Added "No Silver Bullet"

### PR DESCRIPTION
**Pull request by tallesl**
I was surprised that this classic paper wasn't submitted already.
"No Silver Bullet" (along with the great "Out of the Tar Pit") is one of my favourites papers.

I love how he tackled the difficulties and the silver bullets one by one through the paper.
Ending with an ought to design shows how this paper aged so well (in todays world where software development is often just "coding").

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
@tallesl I'm +1 to this, but I've made a comment on the diff that needs changing and a rebase.

File: design/README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
 
+* [No Silver Bullet (1986)](http://faculty.salisbury.edu/~xswang/Research/Papers/SERelated/no-silver-bullet.pdf)
+    > Most of the big past gains in software productivity have come from removing artificial barriers that have made the accidental tasks inordinately hard, such as severe hardware constraints, awkward programming languages, lack of machine time. How much of what software engineers now do is still devoted to accidental, as opposed to essential?
+
 * [Traits: A Mechanism for Fine-Grained Reuse](http://scg.unibe.ch/archive/papers/Duca06bTOPLASTraits.pdf)
 
-* :scroll: [Out of the Tar Pit](out-of-the-tar-pit.pdf)
-    > We identify common causes of complexity and discuss general approaches which can be taken to eliminate them where they are accidental in nature.
\ No newline at end of file
+* [Out of the Tar Pit (2006)](out-of-the-tar-pit.pdf)
```
Comment (zeeshanlakhani) [inline design/README.md]: We want to keep the :scroll: on this, as we have the paper hosted within the directory.

**Comment by zeeshanlakhani** [comment]
@tallesl Mind updating based on my comment?

**Comment by tallesl** [comment]
Oh, I'm so sorry about that, I didn't know (maybe stating on [contributing](https://github.com/papers-we-love/papers-we-love#contributing-guidelines) is a good idea?).

I fixed (and rebased with the current master).

**Comment by zeeshanlakhani** [comment]
Yep, I'll update the contribution section too. Thanks!

**Comment by DarrenN** [comment]
@zeeshanlakhani / @tallesl can we get this up to date with master and merged?

**Comment by DarrenN** [comment]
Closed due to inactivity. Please re-open once the outstanding issues are addressed.

**Pull request closed without merging**